src/components/Quotes.js

```diff
@@ -0,0 +1,58 @@
+import {
+  useEffect, useState, useCallback, useMemo,
+} from 'react';
+import '../styles/quotes.css';
+import Header from './Header';
+
+const Quotes = () => {
```

Comment:
- Optional
I would recommend adding a `loading` state so that when the async operation is still running, you can display the `Loading` text on the screen.